flink-runtime/src/main/java/org/apache/flink/runtime/state/heap/AbstractHeapState.java

```diff
@@ -31,14 +31,13 @@
  * Base class for partitioned {@link State} implementations that are backed by a regular
  * heap hash map. The concrete implementations define how the state is checkpointed.
  *
- * @param <K> The type of the key.
- * @param <N> The type of the namespace.
- * @param <SV> The type of the values in the state.
- * @param <S> The type of State
- * @param <SD> The type of StateDescriptor for the State S
+ * @param <K>	The type of the key.
```

Comment:
All of the generic parameters seem to have a tab in them. That's why they look wonky here. Maybe should fix those.